src/Components/NoMatch/NoMatch.tsx

```diff
@@ -0,0 +1,18 @@
+
+import React from 'react'
+import { Link } from 'react-router-dom'
+import './NoMatch.css'
+
+
+const NoMatch = () => {
+  return (
+    <div className="noMatchContainer">
+      <h1 className="noMatchError">Sorry, page not found!</h1>
+      <Link to='/'>
+          <button>GO TO HOMEPAGE</button>
+      </Link>
+    </div>
+  )
+}
```

Comment:
Good idea implementing this NoMatch component - should make it easier to reuse and keep consistent.